styles.css

```diff
@@ -0,0 +1,214 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap');
+
+@font-face {
+  font-family: Poppins;
+  src: url(./Assets/Poppins/Poppins-Regular.ttf);
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  border-style: border-box;
+}
+
+.homepage {
+  background-image: url(/Assets/header-background-mobile.svg);
+  width: 375px;
+  height: 718px;
+  left: 0;
+  right: 0;
+  font-family: Poppins, sans-serif;
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+}
+
+/* top header css */
+
+nav {
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  margin-top: 50px;
+  margin-left: 7%;
+}
+
+.menu-icon {
+  margin-top: 60px;
+  margin-right: 7%;
+}
+
+.logo-img {
+  width: 76px;
+  height: 20px;
+}
+
+/* secondary header css */
+
+.text-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.main-text {
+  width: 327px;
+  height: 150px;
+  margin-top: 120px;
+  left: 24px;
+  font-weight: 700;
+  line-height: 52px;
+  font-size: 40px;
+  z-index: 1;
+  text-align: left;
+}
+
+.secondary-text {
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 24px;
+  width: 327px;
+  margin-top: 8%;
+  font-style: normal;
+}
```

Comment:
- [ ] Kindly give your `paragraph text` the right color as seen in your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/198144862-d827cfe3-d559-48cd-aef3-e61b482dfd24.png)  |  ![image](https://user-images.githubusercontent.com/74814780/198144759-f004c839-6f46-4ca6-8ccd-a4778876a6f0.png)